src/olympia/bandwagon/views.py

```diff
@@ -121,14 +119,7 @@ def collection_listing(request, base=None):
                   .filter(type=amo.COLLECTION_FEATURED)
                   .exclude(addon_count=0)
     )
-    # Counts are hard to cache automatically, and accuracy for this
```

Comment:
Removed for simplicity, it's only in a legacy view that we should no longer hit anyway.